Huffman Coding/Huffman.playground/Contents.swift

```diff
@@ -3,21 +3,21 @@
 import Foundation
 
 let s1 = "so much words wow many compression"
-if let originalData = s1.dataUsingEncoding(NSUTF8StringEncoding) {
-  print(originalData.length)
+if let originalData = s1.data(using: String.Encoding.utf8) {
```

Comment:
As per our style guidelines, you should shorten this to `s1.data(using: .utf8)`.